lib/v8.js

```diff
@@ -226,6 +227,18 @@ function getHeapCodeStatistics() {
   };
 }
 
+let called = false;
```

Comment:
Would you mind renaming this variable to a more meaningful name? e.g. `isHeapSnapshotNearHeapLimitSet`.